ios/CodePush/CodePushUpdateUtils.m

```diff
@@ -20,6 +20,14 @@ + (void)addContentsOfFolderToManifest:(NSString *)folderPath
     }
     
     for (NSString *fileName in folderFiles) {
+#if TARGET_IPHONE_SIMULATOR
```

Comment:
This file is never generated on iOS's file system. Its only Mac OS X that generates this file. The iPhone simulator (being a simulator, and not an emulator) uses OS X's file system, and hence you see this file on the simulator. So I'm guessing checking for the filename would be useless when the app is being built for release. But if you still insist, I can remove it.